### PR TITLE
Refactor something

### PR DESCRIPTION
**Pull request by ThaddeusJiang**
- package-lock.json was generated for lockfileVersion@2
- fix ESLint error: single quote and semi

### CODE REVIEW

File: src/lib/commands.ts [hunk 1]
```diff
@@ -123,38 +123,38 @@ export function registerCommands(ctx: ExtensionContext, core: Core): Disposable[
               root: workspace.workspaceFolders![0].uri.fsPath,
             },
             { interpretUtilities: true },
-          )
+          );
 
           // CHECK VSCode Theme Color
-          let isDark = true
+          let isDark = true;
```
Comment (alexanderniebuhr) [inline src/lib/commands.ts:129]: no. please change your es-lint config to `non semi`

File: src/lib/core.ts [hunk 2]
```diff
@@ -54,36 +54,36 @@ export async function init(): Promise<Core> {
...
+          default:
+            dynamicCompletions.push({
+              label: utility,
+              position: utility.length - mark,
+            });
+            if (config in negative) {
               dynamicCompletions.push({
-                label: utility,
-                position: utility.length - mark,
+                label: `-${utility}`,
```
Comment (alexanderniebuhr) [inline src/lib/core.ts:82]: why are you doing this change?

File: src/lib/core.ts [hunk 3]
```diff
@@ -54,36 +54,36 @@ export async function init(): Promise<Core> {
...
+                position: utility.length + 1 - mark,
```
Comment (alexanderniebuhr) [inline src/lib/core.ts:83]: why do you need to add 1 here?

**Review by alexanderniebuhr — CHANGES_REQUESTED** [review]
thanks for your work!
all in all not bad.

i know the repo style not very consistent yet, but we do prefer non semicolon style!

also added two comments for you to explain your change

File: src/lib/commands.ts (hunk 1 above)
Comment (alexanderniebuhr) [reply src/lib/commands.ts:129]: all good.. leave it at semi!

File: src/lib/core.ts (hunk 3 above)
Comment (antfu) [reply src/lib/core.ts:83]: It's not `+1`, it's because git diff misalignment. The original line is L83. Same as previous comment

Comment (alexanderniebuhr) [reply src/lib/core.ts:83]: what was the way to show git all in one file like in markdown @antfu

**Review by alexanderniebuhr — APPROVED** [review]

**Comment by ThaddeusJiang** [comment]
> i know the repo style not very consistent yet, but we do prefer non semicolon style!

understood

**Pull request merged**